index.htm

```diff
@@ -24,15 +24,74 @@
     <a href="#" class="button filled-primary-button">Sign-up</a>
   </nav>
   <div class="wrapper">
-      <div id="hero-img">
-        <div id="hero-text">
-          <h1>We Make Your Home Alive</h1>
+    <div id="hero-img">
+      <div id="hero-text">
+        <h1>We Make Your Home Alive</h1>
+      </div>
+      <div id="center-button">
+        <a href="#" class="button filled-primary-button">Shop</a>
+        <a href="#" class="button outlined-secondary-button">Blog</a>
+      </div>
+    </div>
+    <h2 id="gallery-heading">Our Loved Ones</h2>
+    <div id="gallery-container">
+      <div class="gallery-row">
+        <div class="gallery-column">
+          <img src="./res/img/1.jpeg" width="100%">
+          <div class="common-name gallery-box">Common Name</div>
+          <div class="scientific-name" id="scientific-name-color-1">Scientific Name</div>
+        </div>
+        <div class="gallery-column">
+          <img src="./res/img/2.jpeg" width="100%">
```

Comment:
![image](https://user-images.githubusercontent.com/22443504/171016200-1ecb1a34-1210-41cc-af4d-62dbc9dd36a2.png)
This image seems to be not the same height as the others.